.circleci/config.yml

```diff
@@ -16,30 +16,30 @@
 version: 2.1
 
 orbs:
-  python: circleci/python@0.3.2
+  python: circleci/python@2.0.3
 
 jobs:
   python_test:
     parameters:
       python_ver:
         type: string
-        default: "3.6"
+        default: "3.11"
     docker:
       - image: circleci/python:<< parameters.python_ver >>
```

Comment:
`cimg/...` instead of `circleci/...` should fix the 3.10+ build failures

Reply:
I don't see that directory in this repo